add compatibility matrix to the docs (#11660)
RasaHQ/rasa: docs/docs/installation/rasa-pro/installation.mdx

- [Deploying Rasa Plus in production](../../deploy/deploy-rasa-plus)
- [Deploying Rasa Pro Services in production](../../deploy/deploy-rasa-pro-services)

> To learn more about which version of Rasa Pro Services is compatible with Rasa Plus, please head over to the

Reviewer: ```suggestion
To learn more about compatibility between different versions of  Rasa Plus and Rasa Pro Services, please see
```

> [compatibility matrix](../../compatibility-matrix.mdx).

Author: not sure if it's the best place for that plug. LMK!

Reviewer: I think the place is right
